test/jobs/calendar_importer/ld_json_parser_test.rb

```diff
@@ -12,7 +12,7 @@ def check_source_has_events(url, cassette, expected_node_count, expected_event_c
     )
     assert_predicate calendar, :valid?
 
-    VCR.use_cassette(cassette) do
+    VCR.use_cassette(cassette, record: :new_episodes) do
```

Comment:
Oops that shouldn't be there. The tests started failing because our web-blocker was picking up requests it had not seen before and it blocks them and signals an error. So the quick fix is to add the new requests to its little playback file, which is why the `record: :new_epiosdes` flag is present. But you are right to spot that it shouldn't be something left in the repo